### ISSUE TITLE
Enclosing character in CSV export

### ISSUE THREAD

**Issue by arturmeinzer** [zheeeng/export-from-json#116]
Is there a way to change the enclosing character for CSV type. All fields are enclosed in double quote, but I want to change that to an empty string so that they are not enclosed by any character.

**Comment by zheeeng** [comment]
Can you provide a sample data for this?

**Comment by arturmeinzer** [comment]
Yes, here is my sample data:
```
[
            {
                "name": "Palpatine",
                "wallet": "xxx1",
                "payoutAddress": "yyy1",
                "scholar": 0,
                "manager": 0
            },
            {
                "name": "Jane Doe",
                "wallet": "xxx2",
                "payoutAddress": "yyy2",
                "scholar": 46,
                "manager": 47
            },
            {
                "name": "Dugs",
                "wallet": "xxx3",
                "payoutAddress": "yyy3",
                "scholar": 146,
                "manager": 146
            }
        ]
```
And I expect the output to be like this:
```
Name,Wallet,PayoutAddress,ScholarShare,ManagerShare
Palpatine,xxx1,yyy1,0,0
Jane Doe,xxx2,yyy2,46,47
Dugs,xxx3,yyy3,146,146
```
The headline is already like I want it

**Comment by zheeeng** [comment]
You can use `beforeTableEncode` to map over field values to strip enclosing `""`. We have added a tips chapter for it.

BTW, it is not suggested to remove them in case your dataset contains the character `"`.

**Comment by arturmeinzer** [comment]
The function is not helping, because in your converters.js you always enclose in \". I was using a different library before which is called "react-csv". This has an option "enclosingCharacter", but it can only handle csv export. I would like to use only your library, instead of two different ones.

**Comment by zheeeng** [comment]
> The function is not helping, because in your converters.js you always enclose in ". I was using a different library before which is called "react-csv". This has an option "enclosingCharacter", but it can only handle csv export. I would like to use only your library, instead of two different ones.

This option causes the cols spliting bug when source data contains double-quotes. `react-csv` also encounters the same problem. Does it matter for you to determine the format of the intermediate data?

**Comment by arturmeinzer** [comment]
I have no double-quotes in my data, so it would be no problem. Otherwise I can make sure to not have double-quotes in my data before exporting

**Comment by zheeeng** [comment]
Can you explain what is your demand for this option? If it is universal I'd like to support it and add it to README.

**Comment by arturmeinzer** [comment]
I want to use it for the scatter tool for Axie Infinity. There you can paste a list of Ronin Addresses and an an amount separated by comma or empty string, but it is only working without quotes. 
<img width="657" alt="Bildschirmfoto 2022-03-23 um 20 08 40" src="https://user-images.githubusercontent.com/86798022/159777761-3d43fd90-f23d-404c-bd6c-c7a7301cdf35.png">

**Comment by zheeeng** [comment]
Hi, you can try the latest version 1.6, which doesn't contain double quotes when they are necessary.
And there is a patched version v1.5.9, which added an option `encloser`, you can specify it to an empty string.

**Comment by arturmeinzer** [comment]
Nice, thank you!
